test/integration/testpaths/testpath_storage_migration.py

```diff
@@ -479,6 +283,19 @@ def setUpClass(cls):
             )
             cls._cleanup.append(cls.disk_offering_cluster1)
 
+            cls.new_virtual_machine = VirtualMachine.create(
+                cls.apiclient,
+                cls.testdata["small"],
+                templateid=cls.template.id,
+                accountid=cls.account.name,
+                domainid=cls.account.domainid,
+                serviceofferingid=cls.service_offering.id,
+                zoneid=cls.zone.id,
+                mode=cls.zone.networktype
+            )
+
+            cls.new_virtual_machine.start(cls.apiclient)
```

Comment:
Vm is by default in running state as long as we do not pass startvm=False. Hence no need to start VM explicitly after deploying.